src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstBase;
+    private int secondBase;
+    private int heigh;

```

Comment:
There is a typo in the variable name 'heigh'. It should be 'height'. This will improve code readability and prevent potential confusion.